doc/source/computation.rst

```diff
@@ -743,14 +743,15 @@ directly, it's easier to think about either the **span**, **center of mass
     \begin{cases}
         \frac{2}{s + 1},               & s = \text{span}\\
         \frac{1}{1 + c},               & c = \text{center of mass}\\
-        1 - \exp^{\frac{\log 0.5}{h}}, & h = \text{half life}
+        1 - \exp^{\frac{\log 0.5}{h}}, & h = \text{half-life}
     \end{cases}
 
-One must specify precisely one of the three to the EW functions. **Span**
-corresponds to what is commonly called a "20-day EW moving average" for
-example. **Center of mass** has a more physical interpretation. For example,
-**span** = 20 corresponds to **com** = 9.5. **Halflife** is the period of
-time for the exponential weight to reduce to one half.
+One must specify precisely one of **span**, **center of mass**, **half-life**
+and **alpha** to the EW functions. **Span** corresponds to what is commonly
+called an "N-day EW moving average". **Center of mass** has a more physical
+interpretation and can be thought of in terms of span: :math:`c = (s - 1) / 2`.
+**Half-life** is the period of time for the exponential weight to reduce to
```

Comment:
can you add a mini chart showing the allowed domains (use csv-table to show it in markdown)